<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1596</width>
    <height>891</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>OceanZ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="entriesListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="htmlEntriesWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>250</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sheetTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Counter-Strike</family>
      <pointsize>32</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Google Sheet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportTitle">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>800</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Counter-Strike</family>
      <pointsize>32</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>PanCafe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sheetEntries">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Update data from Sheet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportEntries">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Update data from PanCafe</string>
    </property>
   </widget>
   <widget class="QLabel" name="sheetResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>800</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="reportResult">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>200</y>
      <width>800</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sheetTotal">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="reportTotal">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>150</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1596</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEntry">
    <property name="title">
     <string>Entry</string>
    </property>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuEntry"/>
  </widget>
  <action name="actionMatch_Entries">
   <property name="text">
    <string>Match Entries</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
